src/static/settings.html

```diff
@@ -0,0 +1,70 @@
+<html lang="en">
+
+<head>
+  <meta charset="utf8">
+  <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+  <link rel="stylesheet" href="settings.css" type="text/css">
+  <link rel="stylesheet" href="content.css" type="text/css">
+  <link rel="stylesheet" href="hint.css" type="text/css">
+  <link rel="shortcut icon" href="logo/Tridactyl_64px.png" type="img/png">
+  <title>Tridactyl Settings</title>
+</head>
+
+<body class="container">
+  <h1 class="title">Tridactyl Settings</h1>
+  <nav>
+    <ul id="tabbar" class="nav nav-tabs" role="tablist">
+      <li class="nav-item">
+        <a class="nav-link" role="tab" href="#current-settings">Current Settings</a>
+      </li>
+      <li class="nav-item">
+        <a class="nav-link" role="tab" href="#rc-text">.tridactylrc</a>
+      </li>
+      <li class="nav-item">
+        <a class="nav-link" role="tab" href="#about">About</a>
+      </li>
+      <li class="nav-item">
+        <a class="nav-link" role="tab" href="#license">License</a>
```

Comment:
In British English this is spelt Licence. License is commonly understood to mean "legally questionable wall of text that nobody every reads" so we could perhaps allow it.

Apologies for the pedantry.